lib/writer.js

```diff
@@ -182,19 +166,19 @@
       return `${readonly}${it.trivia.base}${it.type}${bracket}${it.trivia.termination};`;
     }
     function callbackInterface(it) {
-      return `${it.trivia.callback}callback${interface_(it)}`;
+      return `${it.trivia.callback}callback${interface_like(it, "interface")}`;
```

Comment:
Could this be?
`${it.trivia.callback}${interface_like(it, "callback interface")}`;

Reply:
That will block emitting this as it is: `callback /* Undesirable but possible comment */ interface {};`

Reply:
ah, right.